sento.asd

```diff
@@ -95,7 +96,9 @@
                  (:file "actor-system-test")
                  (:file "actor-tree-test")
                  (:file "spawn-in-receive-test")
-                 )))
+                 (:file "test-utils")
+                 (:file "message-box-test"
+                     :depends-on ("test-utils")))))
```

Comment:
Since test-utils is defined before if the `depends-on` necessary?

Reply:
No, unless you have `:serial t` option in the `defsystem` form.

Reply:
OK, just add a `serial t` then. In main system we have it as well. I think it's straight forward when the loading mirrors the dependency structure roughly.